cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1486,8 +1545,23 @@ String getPrompt() {
         }
 
         if (isBatchMode()) {
-            buffer.append(" #");
+            String batchPrompt = " #";
+            if (getConfig().isColorOutput()) {
+                batchPrompt = batchPrompt + "] ";
+                batchPrompt = Util.formatBatchPrompt(batchPrompt);
+                return buffer.append(batchPrompt).toString();
+            }
```

Comment:
If color output is not enabled, the `#` is not added into the prompt. Adding of `#` to the prompt shouldn't be dependent  on `color-output` property.

Reply:
I think you mean `]` won't be added but it will. Check line 1567.

Reply:
No, I meant the `#` character. If you run the CLI without colors enabled, the prompt looks:

```
[standalone@embedded /] batch
[standalone@embedded /] /system-property=aa:add(value=bb)
[standalone@embedded /] run-batch
```

However when you run the CLI with colors enabled, the prompt contains extra `#` character. And I think that the `#` should be in the prompt in both cases.

```
[standalone@embedded  /] batch
[standalone@embedded  / #] /system-property=aa:add(value=bb)
[standalone@embedded  / #] run-batch 
```